tst/testbugfix/00069.tst

```diff
@@ -0,0 +1,3 @@
+# 2005/05/05 (TB)
+gap> IsBound( ShowPackageVariables );
+true
```

Comment:
(Might be an idea for a a feature request issue (FRI) / request for enhancement (RFE)